eng/versioning/supported_external_dependency_versions.json

```diff
@@ -101,6 +101,26 @@
     { "com.fasterxml.jackson.module:jackson-module-parameter-names": "2.15.0" },
     { "com.fasterxml.jackson.module:jackson-module-scala_2.12": "2.15.0" }
   ],
+  "jackson_2.16":
+  [
+    { "com.fasterxml.jackson.core:jackson-annotations": "2.16.0" },
+    { "com.fasterxml.jackson.core:jackson-annotations": "2.16.0" },
+    { "com.fasterxml.jackson.core:jackson-core": "2.16.0" },
+    { "com.fasterxml.jackson.core:jackson-databind": "2.16.0" },
+    { "com.fasterxml.jackson.dataformat:jackson-dataformat-avro": "2.16.0" },
+    { "com.fasterxml.jackson.dataformat:jackson-dataformat-xml": "2.16.0" },
+    { "com.fasterxml.jackson.datatype:jackson-datatype-jdk8": "2.16.0" },
+    { "com.fasterxml.jackson.datatype:jackson-datatype-jsr310": "2.16.0" },
+    { "com.fasterxml.jackson.module:jackson-module-parameter-names": "2.16.0" },
+    { "com.fasterxml.jackson.module:jackson-module-scala_2.12": "2.16.0" }
+  ],
+  "reactor_2023":
```

Comment:
nit: any reason why Reactor versions are in descending order while all others are in ascending order?

Reply:
Nope, I didn't even realize that until this was pointed out. I'm going to leave it for now because it really doesn't change anything.